samples/ClassroomLabs/Modules/Library/Tools/hogwarts.csv

```diff
@@ -1,4 +1,4 @@
-ResourceGroupName,Location,LabAccountName,LabName,ImageName,MaxUsers,UsageQuota,UsageMode,SharedPassword,Size,Title,Descr,UserName,Password,LinuxRdp, Emails, idleGracePeriod, idleOsGracePeriod, idleNoConnectGracePeriod, Invitation, Schedules
-hogwarts-rg2,westeurope,Muggle,"History of Magic",CentOS-Based*,2,20,Restricted,false,Basic,"History of Magic","Those who don't know history ...",test0000,Test00000000,true,"lucabol@microsoft.com;lucabol@microsoft.com", 7,8,9,"History of Magic"
-hogwarts-rg2,westeurope,Wizardy,Transfiguration,CentOS-Based*,2,20,Restricted,false,Basic,"Transfiguration","Who are you?",test0000,Test00000000,true,"lucabol@microsoft.com;lucabol@microsoft.com", 7,8,9,"Transfiguration"
-hogwarts-rg2,westeurope,Wizardy,Charms,CentOS-Based*,2,20,Restricted,false,Basic,"Charms","Encanted",test0000,Test00000000,true,"lucabol@microsoft.com;lucabol@microsoft.com",15,0,10, "Charms", charms
\ No newline at end of file
+ResourceGroupName,Location,LabAccountName,LabName,ImageName,AadGroupId,MaxUsers,UsageQuota,UsageMode,SharedPassword,Size,Title,Descr,UserName,Password,LinuxRdp, Emails,LabOwnerEmails, idleGracePeriod, idleOsGracePeriod, idleNoConnectGracePeriod, Invitation, Schedules
+hogwarts-rg2,westeurope,Muggle,"History of Magic",CentOS-Based*,00000000-0000-0000-0000-000000000000,2,20,Restricted,FALSE,Basic,"History of Magic","Those who don't know history ...",test0000,Test00000000,TRUE,"lucabol@microsoft.com;lucabol@microsoft.com","Peter.Hauge@microsoft.com", 7,8,9,"History of Magic"
```

Comment:
@petehauge You changed all the booleans to upper case.  Is there a reason for this?  Just want to make sure we aren't breaking anyone already using the tool that uses lower case true/false.

Reply:
This happened when I opened the CSV file in excel & made changes...  It's not required for the scripts, the casing for PowerShell doesn't matter.  If you want me to switch it back, just let me know - but I figured it would be common that customers open it in excel.  Thanks!

Reply:
No, that's fine.  Just checking.